orttraining/orttraining/test/graph/gradient_graph_builder_test.cc

```diff
@@ -25,6 +25,7 @@ namespace test {
 namespace {
 constexpr auto ORIGINAL_MODEL_PATH = ORT_TSTR("testdata/test_training_model.onnx");
 constexpr auto BACKWARD_MODEL_PATH = ORT_TSTR("testdata/temp_backward_model.onnx");
+constexpr auto PIPELINE_MODEL_BASE = ORT_TSTR("testdata/test_training_model_");
```

Comment:
>("testdata/test_training_model_"); [](start = 45, length = 34)

Intending to add the split models into repo for this test. Searched test_training_model.onnx in github repo but didn't find where it originally comes from. Maybe I missed some folder?